dev-infra/cli.ts

```diff
@@ -27,6 +28,7 @@ yargs.scriptName('ng-dev')
     .command('release <command>', '', buildReleaseParser)
     .command('ts-circular-deps <command>', '', tsCircularDependenciesBuilder)
     .command('caretaker <command>', '', buildCaretakerParser)
+    .command('ngbot <command>', '', buildNgbotParser)
```

Comment:
Since this isn't something we are expecting users to run locally really, lets hide it from the default help menu.

 ```suggestion
    .command('ngbot <command>', false, buildNgbotParser)
```